reader.lua

```diff
@@ -201,10 +201,27 @@ SettingsMigration:migrateSettings(G_reader_settings)
 
 local exit_code
 
+local function getPathFromURI(str)
+    local hex_to_char = function(x)
```

Comment:
```suggestion
    local hexToChar = function(x)
```
That's in our style guide to distinguish functions/methods from variables.